ansible/resources/podman_sc4s.service

```diff
@@ -38,8 +38,8 @@ ExecStart=/usr/bin/podman run \
         -v "$SC4S_ARCHIVE_MOUNT" \
         -v "$SC4S_TLS_MOUNT" \
         --env-file=/opt/sc4s/env_file \
-        --health-cmd="/healthcheck.sh" \
-        --health-interval=10s --health-retries=6 --health-timeout=6s \
+        --health-cmd="/usr/sbin/syslog-ng-ctl healthcheck --timeout 5" \
+        --health-interval=2m --health-retries=6 --health-timeout=5s \
```

Comment:
is it better to change from 10s to 2m?

Reply:
I took it from axoflow https://github.com/axoflow/axosyslog/blob/main/docker/alpine.dockerfile#L123, I think they better know what timeout should be, and you know also copy that on out docker file and it would be strange that on top level we overriding another timeouts

Reply:
<img width="1354" alt="Screenshot 2024-09-26 at 10 48 24" src="https://github.com/user-attachments/assets/6ad43640-dced-4758-a519-de335806bb51">

Reply:
right, I agree